openpype/plugins/inventory/remove_and_load.py

```diff
@@ -0,0 +1,43 @@
+from openpype.pipeline import InventoryAction
+from openpype.pipeline.legacy_io import Session
+from openpype.pipeline.load.plugins import discover_loader_plugins
+from openpype.pipeline.load.utils import (
+    get_loader_identifier,
+    remove_container,
+    load_container,
+)
+from openpype.client import get_representation_by_id
+
+
+class RemoveAndLoad(InventoryAction):
+    """Delete inventory item and reload it."""
+
+    label = "Remove and load"
+    icon = "refresh"
+
+    def process(self, containers):
+        for container in containers:
+            project_name = Session.get("AVALON_PROJECT")
+
+            # Get loader
+            loader_name = container["loader"]
+            for plugin in discover_loader_plugins(project_name=project_name):
+                if get_loader_identifier(plugin) == loader_name:
+                    loader = plugin
+                    break
+
+            assert (
```

Comment:
Loader variable here might be undefined. In this case you might be better of doing a for..else statement.

So you could do like this psuedocode:
```
for plugin in plugins:
    if plugin == plugin_name:
        loader = plugin
        break
else:
    # Failed to get loader
```